src/finch/interface/tensor.py

```diff
@@ -32,3 +32,63 @@ def prod(arr, /, axis=None):
         return lazy.prod(arr, axis=axis)
     else:
         return compute(lazy.prod(lazy.lazy(arr), axis=axis))
+
```

Comment:
don't worry about this one for now, on thursday we'll have a better model for eager arrays.